src/_includes/contributor/rewards.md

```diff
@@ -1,24 +1,37 @@
 
-The Community Engineering team assesses each Pull Request and determines the best awards for the work involved. Contribution Points are calculated according to the assessment results. We award points when the PR is merged.
+The Community Engineering team assesses each PR and determines the best awards for the submitted work. Contribution points are calculated according to the assessment results. We award points when the PR is merged.
 
 ### How points are awarded
 {:.no_toc}
 
-Every merged PR receives one base achievement and potentially one or more additional achievements. These achievements are applied to PRs during review and assessment. Contributors and Maintainers receive points after the PR has been merged.
+Every merged PR receives one [base achievement](#base-achievements) and potentially one, or more, [additional achievements](#additional-achievements). These achievements are applied to PRs during review and assessment. [Contributors](contributors.html) and [maintainers](maintainers.html) receive points after the PR is merged.
 
-Due to the level of work required for developing and reviewing a PR, Contributors and Maintainers receive the same amount of points.
+Due to the level of work required for developing and reviewing a PR, contributors and maintainers receive the same amount of points.
 
-See the following examples for calculated reward points:
+For instance, see this calculated reward points example:
 
--  Contributor submits PR with complex code contributions: Improvement(base) 10 points + Complex(additional) 20 points = 30 points
--  Contributor submits PR with complex code and tests: Improvement(base) 10 points + Complex(additional) 20 points + Test coverage(additional) 10 points = 40 points
--  Contributor submits PR port of existing merged PR:
-   -  Original contributor: (Improvement(base) 10 points + Complex(additional) 20 points) + Author of Ported Issue 5 points = 35 points
-   -  Porting contributor: Port(base) 5 points
--  Maintainer reviews and approves PR with complex code and tests: Improvement(base) 10 points + Complex(additional) 20 points + Test coverage(additional) 10 points = 40 points
+1. When a contributor submits a PR with complex code contributions, they receive a total of 30 points:
+    * Improvement (base achievement) = 10 points
+    * Complex (additional achievement) = 20 points
 
-Earned achievements display as labels on GitHub PRs and for each Magento Contributor, Partner, and Maintainer on [magento.com](https://magento.com/magento-contributors).
+1. When a contributor submits a PR with complex code and tests, they receive a total of 40 points:
+    * Improvement (base achievement) = 10 points
+    * Complex (additional achievement) = 20 points
+    * Test coverage (additional achievement) = 10 points
+1. When a contributor submits a PR port of an existing merged PR, they receive a total of 35 points:
```

Comment:
This section is confusing. Awaiting an SME for clarification

Reply:
I assumed tests are mandatory.

Reply:
Waiting on an EngComm SME to weigh in, as this is determined by them.

Reply:
@VladimirZaets What does the "Porting contributor" line item mean? How does that factor in to the 35 points in this example?

Also, is test coverage mandatory? I notice that there are 10 points allocated here for test coverage.

Reply:
@shrielenee I think we can remove example with "Porting contributor" at all. It was valid when a contributor had the possibility to make backport/upport between different Magento release lines ( 2.2 and 2.3 as an example ). Currently, we have just one branch available for contributions (2.4), so this case is not real anymore and we can remove any mention of it

Reply:
@shrielenee regarding to code coverage. Yes, according to our rules all new code must be covered by tests. But in the case of bugfix, the code can be already covered or sometimes we are covering by our internal resources. Or another example, currently we cannot cover some cleanup fixes or UI fixes. 
We want to review the usage of "test coverage" label, but currently, we add it to PRs that provide code with test coverage

Reply:
Ok, based on this leaving code coverage content in but taking out port contributor content.